packages/bootstrap/scss/card/_variables.scss

```diff
@@ -1,71 +1,184 @@
+/// @group card
 $kendo-card-deck-gap: k-map-get( $kendo-spacing, 4 ) !default;
 
+/// The background color of the Card.
+/// @group card
 $kendo-card-bg: $kendo-component-bg !default;
+/// The text color of the Card.
+/// @group card
 $kendo-card-text: $kendo-component-text !default;
+/// The border color of the Card.
+/// @group card
 $kendo-card-border: $kendo-component-border !default;
+/// The shadow of the Card.
+/// @group card
 $kendo-card-shadow: null !default;
 
+/// The background color of the Card when focused.
+/// @group card
 $kendo-card-focus-bg: null !default;
+/// The text color of the Card when focused.
+/// @group card
 $kendo-card-focus-text: null !default;
+/// The border color of the Card when focused.
+/// @group card
 $kendo-card-focus-border: null !default;
+
+// TODO: double-check: the default theme is not supposed to elevate the Card on focus
+
+/// The shadow of the Card when focused.
+/// @group card
 $kendo-card-focus-shadow: k-elevation(3) !default;
 
+/// The horizontal padding of the Card header.
+/// @group card
 $kendo-card-header-padding-x: $kendo-card-padding-x !default;
+/// The vertical padding of the Card header.
+/// @group card
 $kendo-card-header-padding-y: $kendo-card-padding-y !default;
+/// The bottom border width of the Card header.
+/// @group card
 $kendo-card-header-border-width: $kendo-card-border-width !default;
+/// The background color of the Card header.
+/// @group card
 $kendo-card-header-bg: $kendo-component-header-bg !default;
+/// The text color of the Card header.
+/// @group card
 $kendo-card-header-text: $kendo-component-header-text !default;
+/// The border color of the Card header.
+/// @group card
 $kendo-card-header-border: $kendo-card-border !default;
 
+/// The horizontal padding of the Card body.
+/// @group card
 $kendo-card-body-padding-x: $kendo-card-padding-x !default;
-$kendo-card-body-padding-y: $kendo-card-padding-x !default; // In bootstrap, all card body paddings equal the card-spacer-x var
+/// The vertical padding of the Card body.
+/// @group card
+$kendo-card-body-padding-y: $kendo-card-padding-x !default; // In bootstrap, all card body paddings equal the Card-spacer-x var
 
+/// The horizontal padding of the Card footer.
+/// @group card
 $kendo-card-footer-padding-x: $kendo-card-padding-x !default;
+/// The vertical padding of the Card footer.
+/// @group card
 $kendo-card-footer-padding-y: $kendo-card-padding-y !default;
+/// The top border width of the Card footer.
+/// @group card
 $kendo-card-footer-border-width: $kendo-card-border-width !default;
+/// The background color of the Card footer.
+/// @group card
 $kendo-card-footer-bg: $kendo-card-header-bg !default;
+/// The text color of the Card footer.
+/// @group card
 $kendo-card-footer-text: $kendo-card-header-text !default;
+/// The border color of the Card footer.
+/// @group card
 $kendo-card-footer-border: $kendo-card-header-border !default;
 
+/// The bottom margin of the Card subtitle.
```

Comment:
I think here should be /// The bottom margin of the Card **title**. (please double check)